ballerina-tests/tests/http_client_data_binding.bal

```diff
@@ -445,7 +445,7 @@ service /backend on clientDBBackendListener {
 
     resource function get jsontype() returns http:InternalServerError {
         json j = {id:"hello"};
-        return {body: j};
+        return {body: j, mediaType: mime:APPLICATION_JSON};
```

Comment:
Do we need the mediaType here ?

Reply:
Fixed with [b927caec215436c0c1174ccf51d564e55c8ea4cd](https://github.com/ballerina-platform/module-ballerina-http/pull/839/commits/b927caec215436c0c1174ccf51d564e55c8ea4cd)